bee-network/src/service/controller.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020-2021 IOTA Stiftung
```

Comment:
Nothing in this file is called `*Controller*`, file name is an artefact from the past ?

Reply:
yup. You're right.

Reply:
Removed